Covid19Radar/Covid19Radar/App.xaml.cs

```diff
@@ -56,6 +57,8 @@ protected override void OnInitialized()
             LoggerService.StartMethod();
             LogFileService = Container.Resolve<ILogFileService>();
             LogFileService.SetSkipBackupAttributeToLogDir();
+            BackupAttributeService = Container.Resolve<IBackupAttributeService>();
+            BackupAttributeService.SetSkipBackupAttributeToEventLogDir();
```

Comment:
リファレンスに

>Set this property each time you save a file because some common file operations cause this property to reset to false.

と記載がありますので、対象ファイルの作成後にも設定をしたほうが良いかと思います。

Reply:
ありがとうございます。こちらも修正しましたのでご確認お願いします。
0343d5870851890660702d0d4adf5a8d90c2582c